Sources/PT.PM.TSqlParseTreeUst/TSqlAntlrConverter.cs

```diff
@@ -1071,7 +1072,21 @@ public Ust VisitIndex_option([NotNull] TSqlParser.Index_optionContext context)
             }
             else
             {
-                right = new IntLiteral(long.Parse(context.DECIMAL().GetText()), context.DECIMAL().Symbol.GetTextSpan());
+                string rightText = context.DECIMAL().GetText();
+                var rightTextSpan = context.DECIMAL().Symbol.GetTextSpan();
+                
+                if (int.TryParse(rightText, out int intValue))
```

Comment:
This piece of code gets repetitive and annoying - can it be extracted into separate method?